plugins/MCWhitelist/whitelist_http_api.py

```diff
@@ -0,0 +1,24 @@
+import json
+import requests
```

Comment:
Imports incorrectly formatted. There should be a blank line between stdlib packages and site_packages packages